js/data/locale/afa/langname.jf

```diff
@@ -1,4 +0,0 @@
-{
-    "generated": true,
```

Comment:
Almost all of these deleted languages are actually collective language groups. That is, a name for a group of similar languages. It is okay to delete these.

Reply:
Thank you for the confirmation!